mpu6050/mpu6050.py

```diff
@@ -179,6 +188,12 @@ def set_gyro_range(self, gyro_range):
         # Write the new range to the ACCEL_CONFIG register
         self.bus.write_byte_data(self.address, self.GYRO_CONFIG, gyro_range)
 
+    def set_filter_range(self, filter_range=FILTER_BW_256):
+        """Sets the low-pass bandpass filter frequency"""
+        #TODO - Currently overwrites bits 3,4,5 used for FSYNC pin. Change implementation to fix this. 
+        return self.bus.write_byte_data(self.address, self.MPU_CONFIG,filter_range)
```

Comment:
Hey! First of all thank you for your pull request :) It's been a while since I've modified this code, but I think this should fix your TODO. Can you test if your code still works?

```suggestion
        # Keep the current EXT_SYNC_SET configuration in bits 3, 4, 5 in the MPU_CONFIG register
        EXT_SYNC_SET = self.bus.read_byte_data(self.address, self.MPU_CONFIG) & 0b00111000
        return self.bus.write_byte_data(self.address, self.MPU_CONFIG,  EXT_SYNC_SET | filter_range)
```

Reply:
Thanks for the fix - enabling the filter still works as intended, but I don't have the hardware set up to let me check the sync pins.